Makefile

```
@@ -161,6 +161,21 @@ fmt: ## Check formatting of go sources.
 	@$(MAKE) go.init
 	@$(MAKE) go.fmt
 
+
+YAMLS_DIRS ?= $(ROOT_DIR)/deploy/examples $(ROOT_DIR)/.github/workflows
+YAMLLINT_CONF ?= $(ROOT_DIR)/.yamllint
+
+.PHONY: yamllint
+yamllint:
+	@echo ==== yamllint
+	@yamllint -c $(YAMLLINT_CONF) --no-warnings $(YAMLS_DIRS)
```

Instead of echo-ing a header, we can just run the command and not hide the command that calls yamllint (what `@`) does at the line beginning.
 
```suggestion
	yamllint -c $(YAMLLINT_CONF) --no-warnings $(YAMLS_DIRS)
```